.github/workflows

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ ] [OPTIONAL] Good work including the linters config file, but it's not working, you can consider changing the name of this file to ```linters.yml``` and create a folder ```workflows``` so the structure is like this: ```.github/workflows/linters.yml```

Reply:
Ohh okay thank you I'll try that.

Reply:
@Ismailco After I solved all the comments you told me What's next? Ask for a new review or just merge it sorry today is my second day

Reply:
@wdavidcch, Great work 💯